packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -456,34 +459,66 @@ export default class Meetings extends WebexPlugin {
 
     /**
      * Create a meeting.
-     * @param {string} destination - sipURL, spaceId, phonenumber, meeting link, or locus object}
+     * @param {string} destination - sipURL, hydra spaceId, hydra personId, phonenumber, meeting link, or locus object
      * @param {string} [type] - the optional specified type, such as locusId
      * @returns {Promise} A new Meeting.
      * @public
      * @memberof Meetings
      */
     create(destination, type = null) {
       // TODO: type should be from a dictionary
-      // Check if there is already meeting
-      const meeting = this.meetingCollection.getByKey(SIP_URI, destination);
+      // Create a generic promise for validating if the provided destination is
+      // a hydra personId.
+      let promise;
 
-      if (!meeting) {
-        return this.createMeeting(destination, type)
-          .then((meeting) => {
-            if (meeting && meeting.on) {
-              meeting.on(EVENTS.DESTROY_MEETING, (payload) => {
-                this.destroy(payload.meetingId, payload.reason);
-              });
-            }
-            else {
-              LoggerProxy.logger.error(`meetings->create#ERROR, meeting does not have on method, will not be destroyed, meeting cleanup impossible for meeting: ${meeting}`);
-            }
+      // Attempt to determine if the provided destination is a hydra personId.
+      try {
+        // Deconstruct the provided destination.
+        const deconstructedId = deconstructHydraId(destination);
```

Comment:
The conversation is also present in https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js#L135 I guess its the same one